docs/js/main.js

```diff
@@ -28,6 +28,7 @@ $(document).ready(function() {
   os = getOS();
   updateInstructionsURL(os);
   winUpgradeWarning = $(".windows-upgrade");
```

Comment:
Can we remove the `windows-upgrade` class and just use `windows`. There's no need to have 2 I don't think.

Reply:
the windows-upgrade class creates a warning which at present is not produced by the windows class (unless I misunderstood how it works)

Reply:
I thought that was the intention as the new `<div>` is inside the `upgrade` `<div>` so inherits this anyway.